src/index.html

```diff
@@ -7,10 +7,37 @@
   <title>Seafood</title>
 </head>
 <body>
+  <header class="header">
+    <div class="logo">
+      <a href="#" class="logo-link">
+        <img class="logo-icon" src="" alt="logo-icon">
+      </a>
+    </div>
+    <nav class="navbar">
+      <a class="nav-links" href="#">Home</a>
+      <a class="nav-links" href="#meals">Meals <span id="counter"></span></a>
+      <a class="nav-links" href="#about">About</a>
+    </nav>
+  </header>
   <main>
-    <button></button>
+    <div id="meals" class="cards-container"></div>
   </main>
   <div class="comments"></div>
   <div class="reservation"></div>
+  <footer class="footer">
+    <div class="footer-logo">
+      <img class="logo-icon" src="" alt="logo-icon">
+    </div>
+    <div class="about-us" id="about">
+      <h2 class="contact-header">
+        contant us at :
+      </h2>
+      <div>
+        <a class="footer-links" href="https://www.linkedin.com/in/zeff-adeka-28060820a/"><i class="fa-brands fa-linkedin-in"></i>zeffo96</a>
+        <a class="footer-links" href="https://www.linkedin.com/in/shubham-bhoyar-3337091a7/"><i class="fa-brands fa-linkedin-in"></i>Shubham</a>
+        <a class="footer-links" href="https://www.linkedin.com/in/enis-memic/"><i class="fa-brands fa-linkedin-in"></i>Enis</a>
+      </div>
+    </div>
+  </footer>
```

Comment:
[OPTIONAL] Great way of making the footer look like this but you can also mention Microverse inside the footer.